shiny/ui/datagrid/js/package.json

```diff
@@ -4,13 +4,13 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "build": "tsc -noEmit && node build.mjs",
+    "build": "tsx build.ts",
```

Comment:
I saw you took off `tsc -noEmit`, does `tsx build.ts` do type checking?

Reply:
Oh no, sorry. `tsx` is just a tool to run typescript code without compiling to JS.

I can add back the `tsc -noEmit`.